credit_card_check.py

```diff
@@ -0,0 +1,50 @@
+#! usr/bin/python
+
+#https://docs.cs50.net/2018/x/psets/1/credit/credit.html
+
+#cc_num = "378282246310005"
+
+
+def step_0(cc_num):
+    list_step0 = []
+    for i in range(1, len(cc_num),2):

```

Comment:
Wrong implementation. `starting with the number’s second-to-last digit, and then add those products' digits together.`